### PR TITLE
To address issue #87 Security and privacy targets not defined

### PR DESCRIPTION
**Pull request by sakimura**
To address issue #87

### CODE REVIEW

**Review by Sakurann — CHANGES_REQUESTED** [review]
In my understanding, we are not defining a protocol. we are defining an extension to JWT, a way to represent claims. For detailed comment please see Issue #87

File: main.md [hunk 1]
```diff
@@ -164,6 +164,47 @@ Figure: SD-JWT Issuance and Presentation Flow
 
 # Concepts
 
+From the point of view of the collection limitation, which is one of the main privacy principles, it is desirable to be able to provide only the claims necessary for the verifier to process. In the case of a standard JWT [RFC7519], however, this can only be achieved when the JWT is minted afresh each time and it is not possible to re-use the JWT that is pre-minted for other occasions. For example, a holder may want to re-use the document that contains name, address (municipality, street address, room number), date of birth, and National Identification Number to prove that he is a resident of the municipality so that he can receive the service to the resident. In the case of the normal JWT, this is not possible as the signature is over the entire claim set directly. To achieve it, a JWT that only contains the claim about the municipality needs to be minted.
```
Comment (Sakurann) [inline main.md:167]: How is this text different from what is already in "the introduction"? I would rather have this text as a new "use-cases" section.

Also isn't "data minimization" a more common term than "collection limitation"?

File: main.md [hunk 2]
```diff
@@ -164,6 +164,47 @@ Figure: SD-JWT Issuance and Presentation Flow
...
+A protocol is said to achieve strong uncoverability if the adversary acting as the verifier cannot successfully find the claim values that were not intended to be disclosed to the verifier, with a probability significantly better than negligible. 
+
+## R2. Weak uncoverability
+
+While strong uncoverability is desirable, it is not trivial to achieve it. For example, if the original document was a passport that came with the signature of the national authority, then just by looking at the issuer and its signature, it is trivial to find the nationality of the subject in the release. Also, if the existence of the claim name in the release makes it possible for the verifier to guess the value with a probability significantly better than negligible, then it does not satisfy strong uncoverability. 
+
+With these considerations, weak uncoverability is defined as follows: 
+
+A protocol is said to achieve weak uncoverability if the adversary acting as the verifier cannot successfully find the claim values that were not intended to be disclosed to the verifier from the section of the release that directly corresponds with the claim, with a probability significantly better than negligible.
```
Comment (Sakurann) [inline main.md:183]: I do not understand " from the section of the release that directly corresponds with the claim". the release will contain the claim value of the claim that is being released.
Could this be clarified/paraphrased, please?

File: main.md [hunk 3]
```diff
@@ -164,6 +164,47 @@ Figure: SD-JWT Issuance and Presentation Flow
...
+
+Selective disclosure JWT (SD-JWT) aims to address this issue by introducing an indirection: sign the salted hash of the claims instead of the claim set and provide only the claim value of the claim that is required to be processed by the receiver, which assumes the verifier role. This way, while the claim names and the salted hash of the claim values of all the claims contained in the document issued by the issuer is disclosed, the claim values of other claims are not in principle. 
+
+In this document, a protocol is considered to fulfil the selective disclosure property when it has the following properties: 
+
+## R1. Strong uncoverability
```
Comment (danielfett) [inline main.md:173]: Shouldn't that be "Strong Undiscoverability"?

**Comment by danielfett** [comment]
This has been replaced by PR #225.

**Pull request closed without merging**